src/components/atoms/button/button.scss

```
@@ -0,0 +1,35 @@
+@use "../../variables" as *;
+
+.last-assignment-button {
+  display: block;
+  background-color: $black;
+  transition: background-color 0.2s;
+  color: $white;
+  text-align: center;
+  font-family: "Noto Sans JP";
+  font-size: $fontsize-s;
+
+  @media screen and (min-width: $breakpoint-lg) {
+    width: 200px;
+    height: 55px;
+    padding: 17.5px;
+  }
```

```suggestion
    width: 200px;
    height: 55px;
    padding: 17.5px;
```
デスクトップファーストで書いていく場合、一番大きな想定画面サイズに対するメディアクエリは書かないことが多い気がします。